src/Game.java

```diff
@@ -0,0 +1,91 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Game {
+
+    static int currentPlayer = 0;
+    static boolean gameOver = false;
+    static int numPlayers;
+    static Snake snakes;
+    static Ladder ladders;
+    static List<Player> players;
+
+    public static void main(String[] args) throws IOException {
```

Comment:
why is main method part of Game class?

Reply:
Game class is used as the driver class. The "session" starts here. Hence, the main class.

Reply:
Then it should not contain properties